src/constants/links.ts

```diff
@@ -1,9 +1,26 @@
-export const links = {
-  reaktor: 'https://reaktor.com',
-  twitter: 'https://twitter.com/lauradotjs',
-  youtube: 'https://www.youtube.com/c/LauraBeatris',
-  linkedin: 'https://www.linkedin.com/in/laurabeatris/',
-  podcast: 'https://anchor.fm/usepodcast',
-  github: 'https://github.com/LauraBeatris',
-  blog: 'https://dev.to/laurabeatris'
+type Links = Record<string, { href: string, label?: string }>
```

Comment:
Inforcing a new structure from of the `links` constant, adding a `label` property as a single source of naming for links without having to hardcode on other components